lib/keycloak/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.7'
```

Comment:
this is a good start, please see lib/templates/docker-compose.yml for a reference docker-compose file